internal/database/online/store.go

```diff
@@ -16,8 +17,12 @@ type Store interface {
 	Import(ctx context.Context, opt ImportOpt) error
 
 	// Push streaming feature to online store
+	// Note: Make sure that the table corresponding to the stream feature already exists before executing this method
 	Push(ctx context.Context, opt PushOpt) error
 
+	// Create the stream table for the stream group
+	CreateStreamTable(ctx context.Context, entity *types.Entity, groupID int) error
```

Comment:
if user is to add a new feature to the existing group, should we also use this function to add a new column to the existing table?

Reply:
I won't be using this function, I plan to use a new function `StreamTableAddColumn` to do this

Reply:
hmm maybe combine those two functions? name it as `PrepareTable`?

Reply:
good idea! maybe `PrepareStreamTable` is more suitable?

Reply:
sounds good!

Reply:
done